package.json

```diff
@@ -30,11 +30,13 @@
   "exports": {
     ".": {
       "import": "./dist/index.mjs",
-      "require": "./dist/index.js"
+      "require": "./dist/index.js",
+      "types": "./index.d.ts"
```

Comment:
Surprised to see this works given the insistence in the [documentation](https://github.com/privatenumber/get-tsconfig/pull/29#issuecomment-1175311266) and discussions that "types" **must** come first. Given the ease with which this can be [broken](https://github.com/microsoft/TypeScript/issues/46334#issuecomment-943647545), it'll probably be safer to keep these first as suggested.

Reply:
That's incredible. I've never heard of anyone putting key order constraints on JSON before. I'll swap it over tomorrow, thanks.

Reply:
The key order is crucial part of the exports map specification. TS team deliberately ignored it but most resolvers are flexible enough that ir shouldn’t matter. 

> That said, I'm not sure why tsc doesn't just fall back to the top-level "types" field if all else fails

you would think so but that’d unfortunately make too much sense. They have/had a multi-year system of auto-resolving index.d.ts files within subdirectories (which dequal and many others used) & threw it away for their own hottake on an existing, unowned spec.